src/modules/Category/components/CategoryList.tsx

```diff
@@ -30,7 +31,7 @@ const CategoryList = () => {
     useDepartments(organizationId);
 
   const [departmentId, setDepartmentId] = useState<number | ''>(
```

Comment:
No need of `number | ' '` in type as it will always be a number so you can just keep type as number